### DISCUSSION TITLE
Addition of Stats for Sports

### DISCUSSION THREAD

**Discussion by jayvaglio** [schemaorg/schemaorg#2855] [Ideas]
Is there a schema planned for sports stats?

**Comment by thadguidry** [comment]
It's been loosely talked about before, but Sports stats were decided to be too detailed to be directly added to Schema.org directly.
So instead, several organizations collaborated such as IPTC and others on a layered standard you can participate in here:
https://iptc.org/standards/sportsml-g2/
SportsML 3 is still a sorta work in progress I would say? See for example: http://dev.iptc.org/SportsML
But SportsML 3 already supports RDF/XML which you could begin to use (I think).
The larger question is if any publishers (Google, Bing, Yandex, etc.) outside of News organizations might actually consume and understand any of your provided structured data on the web... that would be something to ask the SportsML folks or Google Webmaster forum, so I would say get involved in their organization and forums.

My hope is that SportsML folks would eventually create a Schema.org extension, but it might not be needed any longer at this point since they seemed to have been embracing RDF more along with XML-Schema and JSON, both of which are machine-readable but with a focus on data services (client/server) and less so on web publishing like Schema.org is with JSON-LD, RDFa, Microdata.
 
**Some Schema.org history**
Here's the old discussion group:  https://groups.google.com/g/sports-schema-collab
The old blog post: http://blog.schema.org/2014/02/schemaorg-sports-vocabulary.html
